Sources/MessagingInApp/Gist/Managers/MessageManager.swift

```diff
@@ -85,6 +88,13 @@ class MessageManager {
     func onReplaceMessage(newMessageToShow: Message) {
         // subclass should implement
     }
+
+    func removePersistentMessage() {
```

Comment:
This function got moved from `ModalMessageManager` into `MessageManager` so that modals and inline can both use it. Function body is unchanged.